CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
```

Comment:
It think it's better that we just describe the change here: https://www.tiny.cloud/docs/tinymce/latest/vue-ref/ we usually don't link in the changelog to things.

Reply:
I think the integrations instantly publishes on merge so add a release version and data here since I don't think we have any other vue related things to release. So add a `## 7.0.0 - 2025-05-02` or similar.

- [ ] Add version to changelog

Reply:
It is published as `rc` on merged. The heading and date can be added when doing the actual release.

Reply:
@tiny-ben-tran  So this means no need to change `##Unreleased` to `## 7.0.0 - 2025-05-02` ?

Reply:
You can leave it as is for now. I will change it when doing the publish